tests/conftest.py

```diff
@@ -225,25 +228,29 @@ def pytest_runtest_setup(item: "pytest.Item"):
             pytest.skip("Skipping old migration test")
 
 
+# adds a custom log location for postgres
+postgresql_proc_with_log = factories.postgresql_proc(startparams=f"--log={pg_logfile}")
```

Comment:
I think we have to quote this to keep the people happy that put white spaces in the names of their files and directories.
```suggestion
postgresql_proc_with_log = factories.postgresql_proc(startparams=f"--log='{pg_logfile}'")
```